main.cc

```diff
@@ -558,7 +558,6 @@ int main(int ac, char** av) {
             sstring api_address = cfg->api_address() != "" ? cfg->api_address() : rpc_address;
             sstring broadcast_address = cfg->broadcast_address();
             sstring broadcast_rpc_address = cfg->broadcast_rpc_address();
-            const auto hinted_handoff_enabled = cfg->hinted_handoff_enabled();
```

Comment:
The change in the behaviour is unacceptable and it relates to my comment for the previous patch in this series.

There is not justification whatsoever in such a change and there is no technical difficulty to preserve the current behaviour.

We should be able to disable any hints activity: both generation and sending; effective immediately as soon as hints were disabled - especially via scylla.yaml.